### PR TITLE
feat: add CreateImportURL method

### PR DESCRIPTION
**Pull request by gumuz**
added new private method CreateImportURL for internal use only

### CODE REVIEW

**Review by peterdeme — APPROVED** [review]
lgtm

File: import.go [hunk 1]
```diff
@@ -46,13 +51,24 @@ type ListImportsResponse struct {
 	Response
 }
 
+// CreateImportURL creates a new import URL.
+// Note: Do not use this.
+// It is present for internal usage only.
+// This function can, and will, break and/or be removed at any point in time.
+func (c *Client) CreateImportURL(ctx context.Context, filename string) (*CreateImportURLResponse, error) {
+	var resp CreateImportURLResponse
+	err := c.makeRequest(ctx, http.MethodPost, "imports_url", nil, map[string]string{"filename": filename}, &resp)
```
Comment (ferhatelmas) [inline import.go:60]: for the correct grammar, it sounds like `import_urls`

File: import.go [hunk 2]
```diff
@@ -46,13 +51,24 @@ type ListImportsResponse struct {
...
+
+	return &resp, err
+}
+
 // CreateImport creates a new import task.
 // Note: Do not use this.
 // It is present for internal usage only.
 // This function can, and will, break and/or be removed at any point in time.
-func (c *Client) CreateImport(ctx context.Context, filename string) (*CreateImportResponse, error) {
+func (c *Client) CreateImport(ctx context.Context, path string) (*CreateImportResponse, error) {
```
Comment (ferhatelmas) [inline import.go:69]: `path` standard package name, might be better to have a different name

Comment (peterdeme) [reply import.go:69]: surprised that the linter doesn't catch this 🤔

Comment (ferhatelmas) [reply import.go:69]: because it's not imported in that file at the moment but it could be later on

Comment (gumuz) [reply import.go:69]: ok, let me think, 'cause filepath is also taken already 😅

Comment (yaziine) [reply import.go:69]: `fp` 😉

Comment (yaziine) [reply import.go:69]: also you need to change in `CreateImportURL` #consistency

**Review by peterdeme — APPROVED** [review]

**Review by ferhatelmas — APPROVED** [review]

**Pull request merged**